exercises/reverse-string/reverse-string.spec.js

```diff
@@ -0,0 +1,33 @@
+var reverseString = require('./reverse-string');
+
+describe('ReverseString', function () {
+  it('empty string', function () {
+    var expected = '';
+    var actual = reverseString('');
+    expect(actual).toEqual(expected);
+  });
+
+  it('a word', function () {
```

Comment:
All `it()` blocks except the first one should be changed to `xit()`.  This allows the student to pass one test at a time.